luma/core/device.py

```diff
@@ -251,6 +255,17 @@ def __toRGB565(self, image):
     def __toRGB(self, image):
         return iter(image.tobytes())
 
+    def __toBGR(self, image):
```

Comment:
i wouldn't mind moving all these helpers to luma.core.util

Reply:
I'm just a bit reluctant to put them in there, because they aren't really intended to be general purpose: 
* The input image is always expected to be an RGB image - making them for public consumption, we'd have to document that the images should be RGB only and/or add an `assert image.mode == 'RGB'` which i don't really want to do in a hot loop
* The return value is an iterator of bytes which is specifically chosen so that the looping around `islice()` is uncomplicated (see https://github.com/rm-hull/luma.core/blob/master/luma/core/device.py#L284). 
* There is nowhere else in the codebase that could readily use these methods AFAICS.